frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx

```diff
@@ -44,16 +51,10 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
                 s.enabledTabs,
                 sidePanelStateLogic.selectors.selectedTab,
                 sidePanelStateLogic.selectors.sidePanelOpen,
-                activationLogic.selectors.isReady,
-                activationLogic.selectors.hasCompletedAllTasks,
+                s.isReady,
+                s.hasCompletedAllTasks,
```

Comment:
Change only semi related to this PR. Noticed when swapping between 3000, we can trigger an error as the logic gets unmounted